src/main/java/com/google/sps/servlets/articlesServlet.java

```diff
@@ -17,11 +17,13 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.lang.*;
```

Comment:
Package java.lang is a default import in java. It doesn't need to be imported.

Reply:
removed on the newest request! :)